static/include/app/apis/generated/data.md

```diff
@@ -1533,17 +1511,9 @@ Add a bounding box to an image specified by its BinaryID.
 
 **Example:**
 
-```python {class="line-numbers linkable-line-numbers"}
-from viam.proto.app.data import BinaryID
-
-MY_BINARY_ID = BinaryID(
-    file_id="<YOUR-FILE-ID>",
-    organization_id="<YOUR-ORG-ID>",
-    location_id="<YOUR-LOCATION-ID>"
-)
-
+```python {class="line-numbers linkable-line-numbers" data-line="5"}
```

Comment:
I know this is auto-generated but it seems like this could be useful and maybe should be kept upstream??

Reply:
It is now just a string, so possibly so. I can make a PR upstream though.

Reply:
https://github.com/viamrobotics/viam-python-sdk/pull/887